src/libs/ck-libs/ampi/ampi.C

```diff
@@ -5960,7 +5960,7 @@ CMI_WARN_UNUSED_RESULT ampiParent* IReq::wait(ampiParent* parent, MPI_Status *st
       complete = true;
       parent->resumeOnRecv = false;
 #if CMK_BIGSIM_CHARM
-      *result = 0;
+      if (result) *result = 0;
```

Comment:
Could maybe add to `int* result /*=nullptr*/` to the function prototype.

Reply:
Done